LibSource/ComplexFloatArray.cpp

```diff
@@ -312,3 +312,20 @@ ComplexFloatArray ComplexFloatArray::create(size_t size){
 void ComplexFloatArray::destroy(ComplexFloatArray array){
   delete[] array.data;
 }
+
```

Comment:
this needs some comments to explain what it does: that it splits real and imaginary onto the first two channels